pkg/arvo/lib/chat.hoon

```diff
@@ -0,0 +1,10 @@
+^?
+|%
+++  ship-from-path
```

Comment:
You might consider putting this into `/lib/chat-store`, too. It isn't doing anything generic with chat data-types, and only implements some logic wrt how chat-store deals with paths.

(Of course, right now, we do this in multiple places, so a `/lib/userspace` or w/e might be justified...)

Reply:
\> /lib/misc

Reply:
/lib/userspace 👍🏻